android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Wed Apr 29 14:45:34 PDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```

Comment:
we probably don't want to upgrade gradle for now. this may break our CI/CD pipeline. can you revert this file?

Reply:
ok i will do it